diff --git a/dist/index.js b/dist/index.js
index 7fc20efc331fe7362e3eca10bf99cac3271e7e79..eab47c38a271c27e14047167407e05d113f9164b 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -219,10 +219,6 @@ var LitElement = class _LitElement extends OriginalLitElement {
     const options = Class.shadowRootOptions;
     const useLightDom = options === noShadowRoot;
     const renderRoot = existingShadowRoot ?? (useLightDom ? this.el : this.el.attachShadow(options));
-    Object.defineProperty(this, "shadowRoot", {
-      // Create shadow root on the proxy instance, to make Lit render content there
-      value: renderRoot
-    });
     if (existingShadowRoot) {
       if (process.env.NODE_ENV === "production") {
         OriginalLitElement.prototype.createRenderRoot.call(this);
